examples/scripts/envSetup.sh

```
@@ -0,0 +1,345 @@
+MINIO_PASS=$minio_pass
+UNOAPI_AUTH_TOKEN=$unoapi_auth_token
+MINIOWEB_SUBDOMAIN=$minioweb_subdomain
+MINIOAPI_SUBDOMAIN=$minioapi_subdomain
+DOCKERNETWORK=$dockernetwork
+POSTGRES_DB=$POSTGRES_DB
+EOL
+
+    echo "Arquivo .env gerado com sucesso."
+
+    # Carregar as variáveis de ambiente do .env
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)